### PR TITLE
added line separation in kubectl page

### LINKED ISSUE #30357: Line separation is needed in `reference/kubectl/overview`
**This is a Bug Report**

**Problem:**
Line separation is needed in [`reference/kubectl/overview`](https://kubernetes.io/docs/reference/kubectl/overview/#in-cluster-authentication-and-namespace-overrides)

[Markdown]
```Markdown
If:
* there is Kubernetes service account token file mounted at
  `/var/run/secrets/kubernetes.io/serviceaccount/token`, and
* the `KUBERNETES_SERVICE_HOST` environment variable is set, and
* the `KUBERNETES_SERVICE_PORT` environment variable is set, and
* you don't explicitly specify a namespace on the kubectl command line
then kubectl assumes it is running in your cluster. The kubectl tool looks up the
namespace of that ServiceAccount (this is the same as the namespace of the Pod)
and acts against that namespace. This is different from what happens outside of a
cluster; when kubectl runs outside a cluster and you don't specify a namespace,
the kubectl command acts against the `default` namespace.
```

[Rendered]
image

**Proposed Solution:**
```diff
If:
* there is Kubernetes service account token file mounted at
  `/var/run/secrets/kubernetes.io/serviceaccount/token`, and
* the `KUBERNETES_SERVICE_HOST` environment variable is set, and
* the `KUBERNETES_SERVICE_PORT` environment variable is set, and
* you don't explicitly specify a namespace on the kubectl command line
+          
then kubectl assumes it is running in your cluster. The kubectl tool looks up the
namespace of that ServiceAccount (this is the same as the namespace of the Pod)
and acts against that namespace. This is different from what happens outside of a
cluster; when kubectl runs outside a cluster and you don't specify a namespace,
the kubectl command acts against the `default` namespace.
```

**Page to Update:**
https://kubernetes.io/docs/reference/kubectl/overview/#in-cluster-authentication-and-namespace-overrides

/kind cleanup
/language en
/good-first-issue

### PR DESCRIPTION
**Pull request by robotjellyzone**
Fixes #30357 

Added line separation in [Kubectl doc](https://kubernetes.io/docs/reference/kubectl/overview/#in-cluster-authentication-and-namespace-overrides) ie [reference/kubectl/overview](https://kubernetes.io/docs/reference/kubectl/overview/#in-cluster-authentication-and-namespace-overrides).

### CODE REVIEW

**Comment by robotjellyzone** [comment]
So sorry for messing up 😅 ! actually I don't know why my editor (atom) removed the spaces from different places

**Comment by jihoon-seo** [comment]
@robotjellyzone How about trying with Visual Studio Code, VI(M), and so on? 😊

File: content/en/docs/reference/kubectl/overview.md [hunk 1]
```diff
@@ -545,4 +546,4 @@ Current user: plugins-user
 
 * To find out more about plugins, take a look at the [example cli plugin](https://github.com/kubernetes/sample-cli-plugin).
 
-
+
```
Comment (jihoon-seo) [inline content/en/docs/reference/kubectl/overview.md:549]: Deleting L549, making only one blank line (L548) exist at the end of file, may be a good change. 😊

```suggestion
```

**Review by jihoon-seo — COMMENTED** [review]
@robotjellyzone PTAL!

File: content/en/docs/reference/kubectl/overview.md (hunk 1 above)
Comment (robotjellyzone) [reply content/en/docs/reference/kubectl/overview.md:549]: Yeah sure and thanks for your review :)

**Comment by jihoon-seo** [comment]
- This PR resolves #30357
- This PR is similar with the PR #30425
- This PR is similar with the PR #30551

**Review by Arhell — COMMENTED** [review]
/lgtm

**Pull request merged**